assets/scss/6-components/checkbox/_checkbox.scss

```diff
@@ -0,0 +1,16 @@
+// Checkbox (c-checkbox)
+//
+// A good old-fashioned checkbox with light styling for alignment.
+//
+// Markup: 6-components/checkbox/checkbox.html
+//
+// Styleguide 6.1.5
+.c-checkbox {
+  line-height: $line-height-b;
+
+  input {
+    margin-bottom: 2px;
+    margin-right: 2px;
+    vertical-align: middle;
+  }
+}
```

Comment:
This is a matter of opinion, but the styles added here don't change the look and feel that drastically. This might sound lazy, but contrary to what I've said below about :focus, could this be a case where the browser styles _are_ acceptable?

I did some default browser style digging and the margins are all around this give or take a pixel...
Chrome: margin-bottom: 3px; margin-right: 2.89px;
FF: margin-bottom: 2px; margin-right: 2px;
Safari: margin-bottom: 3px; margin-right: 2px;
Safari: margin-bottom: 3px; margin-right: 2px;
Edge: margin-bottom: 3px; margin-right: 3px;

Toggling the line-height setting doesn't seem to change much either and I almost prefer no vertical-align: middle.

Reply:
I mostly agree, but I do think the line-height is important. In UMP, I have a long label that looks pretty mashed without line height. See below. I opted for putting this rule directly on a `c-checkbox` class because I wasn't sure the `t-space-heading-*` classes would be around much longer given you put that line-height directly on `p` tags. Open to any suggestions here.

Reply:
Yeah, I agree about the line-height. Not sure if this is possible, but can the text be indented so it doesn't wrap beneath the checkbox? I think that would look cleaner, especially if we have multiple checkboxes in a row. 
![Screen Shot 2019-08-08 at 3 44 35 PM](https://user-images.githubusercontent.com/2700000/62736650-7608f200-b9f3-11e9-9374-42cf7232e6ab.png)

Separate from that, is it possible to align the text more with the checkbox? It looks pretty good in the Queso view, but when I click over to "standalone" it feels a little wonky.

Reply:
@emilyyount Aligning checkboxes w/ text is a game of whack-a-mole. Each browser puts different default margins around them, and each one has a different size rendering of the checkbox itself. I favor letting browser default styles handle it.

Reply:
FWIW I've been still sprinkling in t-space-heading-* at times so I'm not counting on it going away. There's also `t-space-base` which gives that 1.4 val.

@emilyyount the discrepancy you speak to might be on me. The standalone view parses the html and only includes the CSS the HTML references. I made a separate CSS file without the normalize resets to make it more concise, but that's misleading now that I think about it. @AndrewGibson27, I'm going to push a fix for that.

Reply:
@emilyyount also the preview on the component page is also throwing in some bulma rules 😬

Reply:
@AndrewGibson27 and now I see why we need the margins, normalize so kindly [removes all margins](https://github.com/texastribune/queso-ui/blob/master/assets/scss/3-resets/_normalize.scss#L259). Not sure I love that, but it all makes sense now.

Reply:
Thanks, y'all! That makes sense.

@ashley-hebler Let me know when that fix is in. I had another comment about the mini form, but it's related to it looking different in the Queso view and the standalone view, so that fix might negate my question :)

Reply:
This seems to play well across browsers. Gonna ship it unless y'all say otherwise.
![screen_shot_2019-08-08_at_4 30 12_pm](https://user-images.githubusercontent.com/2243202/62740198-be78dd80-b9fc-11e9-9927-f0636c002e05.png)

Reply:
I ❤️ it